pkg/client/client.go

```diff
@@ -961,6 +971,7 @@ func monitorKeepTerminatedEvent(
 	operatorPublicKey *operator.PublicKey,
 	keep chain.BondedECDSAKeepHandle,
 	keepsRegistry *registry.Keeps,
+	derivationIndexStorage *recovery.DerivationIndexStorage,
```

Comment:
We'll have to refactor `client.go` once the integration branch changes are merged to `master`. We have been postponing it for too long and now we have 13 parameters in a function and pass these parameters through the entire stack of function calls. 😓 We should do this refactor a couple of months ago.

Reply:
![image](https://user-images.githubusercontent.com/1045160/115417752-e2775500-a1c6-11eb-9579-7685b029b83c.png)